picard/track.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, id, album=None):
     def __repr__(self):
         return '<Track %s %r>' % (self.id, self.metadata["title"])
 
+    def __lt__(self, other):
```

Comment:
__lt__ and __le__ were implemented, I am leaving __eq__ as python does a default id(obj) for that, which is what is the intended behaviour for these items currently.